ca_gazebo/launch/create_april_tags.launch

```diff
@@ -12,6 +12,7 @@
 	<arg name="gui" value="$(optenv GUI true)"/>
 	<arg name="debug" value="$(optenv DEBUG false)"/>
 	
+	<env name="GAZEBO_MODEL_PATH" value="$(find ca_gazebo)/models"/>
```

Comment:
Is this necessary? The best way of doing it is working [here](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_gazebo/package.xml#L24-L28).